### PR TITLE
chore: 7.0.1 release notes typo fix and clarification

### PR DESCRIPTION
**Pull request by ecormany**
# Overview

Fixing up some mistakes in the release notes found in alpha 5 update flow.

# Test Plan

Read the markdown words very carefully.

# Changelog

- say `7.0.1` instead of `7.0.0` in the section about 7.0.1 🤪 
- fix up which versions have which `opentrons_simulate` capabilities. we had some PRs crossing paths on this 🔀 

# Review requests

Please please please make sure I don't have any more dumb typos.

# Risk assessment

None, embarrassment potential only.

### CODE REVIEW

**Review by nusrat813 — APPROVED** [review]
✅

File: api/release-notes.md [hunk 1]
```diff
@@ -14,10 +14,7 @@ This update may take longer than usual if you are updating from v6.x. Allow **ap
 
 ### Known Issues
 
-Some protocols can't be simulated with the `opentrons_simulate` command-line tool:
-
-- JSON protocols created or modified with Protocol Designer v6.0.0 or higher.
-- Python protocols specifying an `apiLevel` of 2.14 or higher.
+JSON protocols created or modified with Protocol Designer v6.0.0 or higher can't be simulated with the `opentrons_simulate` command-line tool.
```
Comment (jwwojak) [inline api/release-notes.md:17]: You could flip the sentence around and get out of the passive voice ("be simulated").

```suggestion
The `opentrons_simulate` command-line tool can't simulate JSON protocols created or modified with Protocol Designer v6.0.0 or higher.
```

Either way is fine. Just an observation for consideration.

File: app-shell/build/release-notes.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ log][]. For a list of currently known issues, please see the [Opentrons issue tr
 
 ## Opentrons App Changes in 7.0.1
 
-Welcome to the v7.0.0 release of the Opentrons App! This release builds on the major release that added support for Opentrons Flex.
+Welcome to the v7.0.1 release of the Opentrons App! This release builds on the major release that added support for Opentrons Flex.
```
Comment (jwwojak) [inline app-shell/build/release-notes.md:11]: Would it help clarity if "the major release" had more specificity -- by working in the version?

"This release builds on the _insert version number here_ that added support for Opentrons Flex."

**Review by jwwojak — APPROVED** [review]
LGTM and added a couple of very optional suggestions.

File: api/release-notes.md (hunk 1 above)
Comment (ecormany) [reply api/release-notes.md:17]: leaving. it's parallel with the text in the 7.0.0 section (which needed the passive to introduce multiple bulleted objects)

File: app-shell/build/release-notes.md (hunk 2 above)
Comment (ecormany) [reply app-shell/build/release-notes.md:11]: i've already gotten into trouble with numbers once 😅 

i think it's better to have one and only one version number in this sentence.

**Pull request merged**